benchmarks/polys.py

```diff
@@ -39,7 +39,10 @@ def to_poly(self, expr):
         return Poly(expr, self.syms)
 
     def to_ring(self, expr):
-        return self.ring(expr)
+        try:
+            return self.ring(expr)
+        except:
+            return expr
```

Comment:
Bare except should never be used.

Anyway this is fixing the problem in the wrong place.